gitcommands.go

```diff
@@ -423,6 +423,10 @@ func gitSquashPreviousTwoCommits(message string) (string, error) {
 	return runDirectCommand("git reset --soft HEAD^ && git commit --amend -m \"" + message + "\"")
 }
 
+func gitRebaseTwoCommits(shaValue string, action string) (string, error) {
+	return runDirectCommand("GIT_SEQUENCE_EDITOR='sed -i \"s/^pick " + shaValue + "/" + action + " " + shaValue + "/g\" $1' git rebase -i " + shaValue + "~2")
```

Comment:
Is there a way we can do this without depending on `sed`? As annoying as it is, we want to maintain compatibility with windows

Reply:
It will take some time. I will try.

Reply:
There is no anything like streaming editor on Windows.
I think a simple file editor program is needed for `git rebase -i` operation.
Should we have another program to help lazygit, something like lazygit-reword?